src/signal_wrap.cc

```diff
@@ -64,6 +64,15 @@ class SignalWrap : public HandleWrap {
   static void Start(const FunctionCallbackInfo<Value>& args) {
     SignalWrap* wrap = Unwrap<SignalWrap>(args.Holder());
     int signum = args[0]->Int32Value();
+#if defined(__POSIX__) && __POSIX && HAVE_INSPECTOR
```

Comment:
Why the `__POSIX`? It’s not defined for me, and the block here is not compiled even though I’m definitely on a POSIX platform.